Automattic/wp-calypso
# Segmentation Survey: Implement save answers mutation and localStorage

Closes https://github.com/Automattic/wp-calypso/issues/89087

## Proposed Changes

* Create mutation to save answers
* Create a hook to handle the answers on the local storage
* Apply mutation and query to the main components

## Testing Instructions

* Apply this PR to your local
* Revert the last commit to access the testing route `git revert 3b18189`
* Go to http://calypso.localhost:3000/ecommerce-segmentation-survey?flags=ecommerce-segmentation-survey
* Fill in the question and click Continue. Check on the network tab if the save request was made
* Follow the instructions here D144372-code to verify if the data was properly saved
* Skip the next question, the save request shouldn't be performed and no data saved on the database (even if you fill in and the data was saved on local storage)

## Pre-merge Checklist
